packages/composables/src/useProductAssociations.ts

```diff
@@ -53,16 +53,16 @@ export function useProductAssociations(
     searchParams: ShopwareSearchParams;
   }) => {
     isLoading.value = true;
-    const method = params.method || "get";
+    const method = params?.method || "get";
```

Comment:
Params are obligatory. Only `method` can be undefined. We need to trust more in TS :)